components/monitoring/prometheus/base/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- install-prometheus.yaml
+- prometheus-view.yaml
+- configure-prometheus.yaml
+- prometheus-servicemonitors.yaml
+
+namespace: "appstudio-workload-monitoring"
```

Comment:
May be we should remove this namespace line. as this by default changes all the namespaces to monitoring namespace for all the components in the base directory. We may need to exclude the servicemonitors from this.

Reply:
I thought ServiceMonitors will be provided by operators in their own namespace, so we would not need the `prometheus-servicemonitors.yaml` at all?

Reply:
we need that for scrapping prometheus it-self. But as we are already putting namespaces in the yamls. we can remove it anyway

Reply:
as discussed on Slack, we'll keep this transformer for now.